modules/mono/glue/runtime_interop.cpp

```diff
@@ -84,7 +86,8 @@ void godotsharp_stack_info_vector_destroy(
 void godotsharp_internal_script_debugger_send_error(const String *p_func,
 		const String *p_file, int32_t p_line, const String *p_err, const String *p_descr,
 		bool p_warning, const Vector<ScriptLanguage::StackInfo> *p_stack_info_vector) {
-	EngineDebugger::get_script_debugger()->send_error(*p_func, *p_file, p_line, *p_err, *p_descr,
+	const String file = "res://" + path::relative_to(*p_file, ProjectSettings::get_singleton()->get_project_data_path());
```

Comment:
What is this needed for? Would it be possible to do this only for paths that are actually part of the project? Clicking one of the stack frames should open that location in the text editor. Currently, this works with files outside the project as well. It would be a pity to lose that.

Reply:
Honestly, this can be scrapped out of the PR. In my humble opinion, this should be handled on the editor side instead. Where we shouldn't just hard-code **C++ Source**.

Reply:
I think this would do the job:

```suggestion
	const String file = ProjectSettings::get_singleton()->localize_path(p_file->simplify_path());
```

Reply:
Tested with files inside and outside `res://` and it works fine.